torch/distributed/fsdp/wrap.py

```diff
@@ -181,20 +198,28 @@ def _recursive_wrap(
     assert wrapper_cls is not None, "Must specify wrapper_cls"
     # Make sure no child is already wrapped.
     for _, child in module.named_modules():
+        if child in ignored_modules:
```

Comment:
I think intuitively ignoring a module means that all its children should be ignored, just want to make sure these changes adhere to this?